docs/DeveloperGuide.md

```diff
@@ -111,6 +111,112 @@ ___
 
 This section describes the implementation details of the features within Property Rental Manager.
 
+---
+### Add Feature
+The add feature simply adds an entity to its corresponding list. For Property Rental Manager, there are two variations to the add feature, namely `add -client` and `add -property`.
```

Comment:
Can try to avoid word such as "simply" in this context because it sounded quite informal.